util/rlp/fuzz/fuzz_targets/append_raw.rs

```diff
@@ -0,0 +1,18 @@
+#![no_main]
+#[macro_use] extern crate libfuzzer_sys;
+#[cfg(feature = "nightly"])
```

Comment:
shouldn't it be `#![cfg(feature = "nightly")]` to be applied to the whole source file? to me this looks like it will only import the `rlp` crate on nightly, but still try to do everything else on stable.
same for `untrusted.rs`